docs/_markbind/navigation/userGuideSections.md

```diff
@@ -0,0 +1,11 @@
+<markdown class="website-content">
+- ## Sections
+- <a href="{{baseUrl}}/userGuide/userQuickStart.html">Quick Start</a>
+- <a href="{{baseUrl}}/userGuide/developingASite.html">Developing a Site</a>
+  - <a href="{{baseUrl}}/userGuide/siteConfiguration.html">Site Configuration</a>
+- <a href="{{baseUrl}}/userGuide/contentAuthoring.html">Content Authoring</a>
+  - <a href="{{baseUrl}}/userGuide/includingContents.html">Including Contents</a>
+- Deploying a Site
+  - <a href="{{baseUrl}}/userGuide/ghpagesDeployment.html">Github Pages Deployment</a>
+  - <a href="{{baseUrl}}/userGuide/netlifyPreview.html">Preview site on Netlify</a>
+</markdown>
```

Comment:
Quick question: why are we not using markdown syntax for links here? Also, let's use `* item` syntax for lists, to be consistent with other places.

Reply:
Not sure, I tested it using markdown syntax works fine. Will change it.

> Also, let's use * item syntax for lists, to be consistent with other places.

👍